src/components/Modals/ChangeLocationModal.vue

```diff
@@ -81,6 +76,19 @@ export default {
     },
     torrent() {
       return this.getTorrent(this.hash)
+    },
+    isPhone() {
+      return this.$vuetify.breakpoint.xsOnly
+    },
+    dialog: {
```

Comment:
Why copy-paste this code for every single modal? That's the whole point of the Modal Mixin

Reply:
because of closing animation.

Reply:
Yes, just increase the delete timeout a little bit in the mixin. This copy/pasting for the dialog introduces a lot of extra boiler plate code